app-misc/dfshow/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>pepitsizous@zoho.com</email>
+	<description>Primary maintainer</description>
+</maintainer>
+<longdescription>DF-SHOW (Directory File Show) is a Unix-like rewrite of some of the applications from Larry Kroeker's DF-EDIT (Directory File Editor) for MS-DOS and PC-DOS systems, based on the Version 2.3d release from 1986.</longdescription>
```

Comment:
Please check the proper indentation altogether in this metadata file. This can be splitted in multiple lines, and wrapped at around 80 chars.
```
<longdescription>
text goes here
more text
end.
</longdescription>
``` 
with proper indentantions of course.